x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/bulk_actions.tsx

```diff
@@ -148,12 +148,21 @@ export const AgentBulkActions: React.FunctionComponent<{
     },
   ];
 
+  const showSelectEverything =
+    selectionMode === 'manual' &&
+    selectedAgents.length === selectableAgents &&
```

Comment:
`selectableAgents` is only on the current page right? so you can select everything but have non selectable agents on other page?

Reply:
I think that's correct. The current behavior shouldn't change. This expression was just moved from and named .